src/FSharp.Management.PowerShell/TypeInference.fs

```diff
@@ -39,7 +39,11 @@ let private getOutputTypesFromSharePointCmdlets(cmdlet:CmdletConfigurationEntry)
         |> Seq.fold (fun state value -> state || value) false
     if (not <| isSPassembly) then None
     else
-        match baseTy.GenericTypeArguments with
+        let types = 
+            baseTy.GetGenericArguments()
+            |> Seq.filter (fun t -> not t.IsGenericParameter)
```

Comment:
@sergey-tihon, thanks for the review.

I made this change based on those documentations:

https://msdn.microsoft.com/en-us/library/system.type.generictypearguments%28v=vs.110%29.aspx
https://msdn.microsoft.com/en-us/library/system.type.getgenericarguments%28v=vs.110%29.aspx

If you think an open generic type can be passed in this statement:

```fsharp
let baseTy = cmdlet.ImplementingType.BaseType
```

then the code would not be correct.

I assume it's never the case but I'm not at all aware of Sharepoint commandlets.

Could anyone having the right environment test this specific change with a Sharepoint commandlet based on a generic type?

Thanks.